# Output DLLs to different directories so that we don't have a clash when
# they're merged back together

--- configure.orig
+++ configure
@@ -7320,7 +7320,12 @@
 
 
 
-dlldir="\${libdir}/wine"
+if test "x$enable_win64" = "xyes"
+then
+  dlldir="\${libdir}/wine64"
+else
+  dlldir="\${libdir}/wine"
+fi
 
 DLLFLAGS="-D_REENTRANT"
 
